tests/foreman/api/test_errata.py

```diff
@@ -29,6 +29,8 @@
 from robottelo.cli.factory import setup_org_for_a_custom_repo
 from robottelo.cli.factory import setup_org_for_a_rh_repo
 from robottelo.config import settings
+from robottelo.constants import DEFAULT_ARCHITECTURE
```

Comment:
The constants module is already imported, please reference these variables through `constants.DEFAULT_ARCHITECTURE` and `constants.DEFAULT_SUBSCRIPTION_NAME`.

`DISTRO_RHEL7` reference could also be cleaned up here, but isn't directly related to your changes.

Reply:
OK, fixed those